css/types/ratio.json

```diff
@@ -31,13 +31,13 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": "4.1"
             },
             "safari_ios": {
-              "version_added": true
+              "version_added": "4.2"
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "1.0"
             },
             "webview_android": {
               "version_added": true
```

Comment:
Is the decision around <= settled enough that we should start changing true values in PRs?

Reply:
It's not yet merged.